models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_curve_pools.sql

```diff
@@ -27,11 +27,7 @@ select explode(sequence(to_date('{{ project_start_date }}'), now(), interval 1 d
         decimals,
         avg(price) AS price
     FROM {{source('prices','usd')}}
```

Comment:
then i think you're right here, remove incremental filters throughout all other CTEs in order to get full historical aggregations on each day